c/build_all/linux/setup.sh

```diff
@@ -12,7 +12,7 @@ repo_name_from_uri()
 }
 
 scriptdir=$(cd "$(dirname "$0")" && pwd)
-deps="curl libcurl4-openssl-dev uuid-dev uuid g++ make cmake git"
+deps="curl libcurl4-openssl-dev uuid-dev uuid g++ make cmake unzip openjdk-7-jre"
```

Comment:
I'm curious...if we download .tar.gz from GitHub instead of .zip (https://github.com/dcristoloveanu/qpid-proton/tarball/0.9-IoTClient), will that save us from having to install an extra component?  'tar' is available in Ubuntu by default, but not sure about Raspbian...?